attivita/viste.py

```diff
@@ -334,7 +342,10 @@ def attivita_scheda_cancella(request, me, pk):
     if not attivita.cancellabile:
         return errore_generico(request, me, titolo="Attività non cancellabile",
                                messaggio="Questa attività non può essere cancellata.")
-
+    
+    gruppo = Gruppo.objects.filter(attivita=attivita)
```

Comment:
Non credo che questo sia richiesto: alla cancellazione dell'attivita', non vogliamo che il gruppo venga cancellato automaticamente (o, perlomeno, senza che questa cosa venga detta esplicitamente all'utente).

Credo che l'unico caso in cui questo comportamento potrebbe essere accettabile sia ne caso in cui il gruppo sia vuoto (non avente membri).

Reply:
> Se un'attività viene cancellata e quindi di fatto non esiste più, il gruppo associato ad essa (gruppo specifico) perchè dovrebbe esistere ancora?

Perche' il gruppo di lavoro potrebbe continuare ad essere utile oltre la vita della singola attivita' (ad esempio, per coadiuvare l'organizzazione di una nuova attivita' simile a quella cancellata). 

Per intenderci, l'unica cosa che non mi piace e' cancellare il gruppo "silenziosamente". Se chiedessimo conferma all'utente ("Vuoi cancellare anche il gruppo di lavoro legato all'attivita'?"), allora non mi preoccuperei.

Pero', sicche' abbiamo la possibilita' di cancellare un gruppo di lavoro (dopo che l'attivita' e' stata cancellata), suggerirei di prendere la strada piu' facile e semplicemente lasciare l'onere di cancellare il gruppo all'utente, se lo desidera.

>  Se non viene cancellato c'è un altro problema:
quando l'attività viene cancellata il campo attività per i gruppi specifici viene settato Null. Essendo Null, il gruppo specifico verrà visualizzato tra i gruppi disponibili come se fosse un gruppo creato per un'attività non specifica (ovvero per un'area).

Quella che descrivi mi sembra una limitazione accettabile per ora.